### DISCUSSION TITLE
ANURA3D_2023 error calculation

### DISCUSSION THREAD

**Discussion by Aidamehr** [Anura3D/Anura3D_OpenSource#186] [General]
Dear developers,
I recently tried to install ANURA3D_2023, but in providing the GID_problemtype, two files are missing: ANURA3d_2023.mat and ANURA3d_2023.prb. I am afraid this is the reason I get this error:

where can I find these files? because they are not in the GitHub directory
![Error](https://github.com/Anura3D/Anura3D_OpenSource/assets/90531027/52c44e05-3f10-4dd4-93a0-24b74d8222dd)

**Comment by murphy-john** [comment]
This appears to be an error where some information was not added to the GOM file. Did you follow the directions in the tutorial manual exactly? It appears you added the EXE file to the *.GiD folder instead of the *.A3D folder.

**Comment by CH-max92** [comment]
Hi Aidamehr. I had the same problem recently,and I found a possible solution after many attempts. The reason that cause this might be the wrong version of the GiD software. I suggest you use GiD **ver.16.0.2** to try it again. Hopefully my answer could solve your problem that also bothored me before.

**Reply by aube260** [reply]
Thank you for posting this. I had installed GiD 16.0.7 and had been having trouble creating the GOM and CPS files (empty). In troubleshooting I had tried multiple different installs of VS etc. Rolling back to GiD 16.0.2 was the fix. Thank you!
